### ISSUE TITLE
Space Biome

### ISSUE THREAD

**Issue by SubjectEleven** [micdoodle8/Galacticraft#1363]
This is just a minor Bug I believe, not really a thing to worry about.

When I go to my Spacestation and press F3, the Biome indicated is Plains. It only changes into Space when I go further into Space by myself, thus leaving the ship behind. When the Biome finally changes to Space, I return to my Spacestation and no sign of Plains Biome anymore. So the bug is that it still keeps saying Plains, everytime I log in and find myself in the Spacestation.

I am using EnviroMine and I am always up in my Spacestation(so I wanted to spicen it up a little, by adding the Spacebiome to the Config of Enviromine). So each time I find myself in the Spacestation it says Plains(which is quite warm), while it should normally say Space(which, I believe, is colder than a Plains Biome).  So yeah, this paragraph is just to explain why it is not as convenient for me(the bug itself has nothing to do with the EnviroMine mod).

TL;DR: It says Plains Biome in the Spacestation instead of Space.

**Comment by radfast** [comment]
It won't break anything.  It's probably linked with #1090 or #1190.

Which other mods do you have installed?  Biomes o' Plenty?  Please give the whole modlist, or better still paste up your fml-client-latest.log somewhere for me to look at.

**Comment by SubjectEleven** [comment]
Yeah, of course it won't really break anything, but it's just that I get the wrong temperature then.
The Mods installed are:
Carpenter's Blocks
Chisel
CodeChickenCore
CodeChickenLib
EnviroMine
Ex Nihilo
ForgeMultipart
galacticraft Core
Galacticraft Planets
HQM-The Journey
Mantle
MicDoodleCore
Minechem
Mekanism
MrTJPCore
NEI
ProjectRed(Base, Compat, Integration, Lighting, Mechanical and World)
And Tinker's Construct

for the log? Just %appdata%, Roaming, Minecraft, Logs, and then the latest log?

**Comment by radfast** [comment]
Yes please, the log should be called fml-client-latest

I'm interested in whether any of those other mods has any non-vanilla biomes, because the usual cause of this type of issue is a biome ID conflict

**Comment by radfast** [comment]
Closing now due to lack of response in 14 days.

I have tested this issue and I don't see it with only Galacticraft installed.  As stated already, I think the cause must be a biome ID conflict from one of those other mods.  Without checking through all of their code it's hard to know which, but the full log might have clues...
